tests/example-czechitas.spec.ts

```diff
@@ -10,12 +10,16 @@ test.describe("Example test class for functionality showcase", () => {
     await pages.visitPage();
   });
 
-  test("contactsPageUrlTest", async () => {
+  test("When user navigates to Contacts section then user is navigated to czechitas url", async () => {
     await pages.headerMenu.goToContactsSection();
-    await pages.mainPage.checkPageUrl("www.czechitas.cz");
+
+    const actualPageUrl = await pages.mainPage.pageUrl.textContent();
+    const expectedPageUrl = "www.czechitas.cz";
+
+    await expect(actualPageUrl).toBe(expectedPageUrl);
```

Comment:
Not sure if this is "low code".

Reply:
Well, yes but let's agree on how do we see low-code nowadays. This way it's definitely more readable. But that just my PoV readability > low-code
Wdyt?

Reply:
I am worry about complexity of that. They doesn't have any programming skill, so it would be hard to explain another concept. 
I don't know, I just want to avoid some problems and questions "Why does this work?" etc. It is not wrong question, but I don't think that we will have time for that.

So it would be just easier to put everything to page object and say "use it". 

I don't think that we have time to go deeper.

Reply:
Okay.